src/NexusMods.DataModel/FileHashCache.cs

```diff
@@ -131,6 +131,7 @@ await IndexFoldersAsync(installation.LocationsRegister.GetTopLevelLocations().Se
             DiskStateEntry.From(h))));
     }
 
+    [SkipLocalsInit] // We don't need to zero the memory here
     private void PutCachedAsync(AbsolutePath path, FileHashCacheEntry entry)
     {
```

Comment:
Since you're already editing the file, might as well rename this to `PutCached`. There is nothing async about this.